test/studies/stencil9/stencil9-explicit.chpl

```diff
@@ -19,24 +20,26 @@ const LocDom = {1..n  , 1..n  },
 const LocaleGridDom = Dom._value.dist.targetLocDom,
       LocaleGrid = Dom._value.dist.targetLocales;
```

Comment:
Oh, good catch!  I didn't revisit the code in enough detail to have thought of that (but wanted to point one of the intended readers in the direction of those features, so this is a great change to make).